07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,74 @@
+.portalnavbar {
+    background-color: #313131;
+    color: white;
+    font-family: Tahoma;
+    height: 80px;
+    width: 1100px;
+    padding-top: 30px;
+    padding-bottom: 30px;
+    padding-left: 110px;
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+   box-sizing: border-box;
+
+}
+.professionheader {
+  font-family: Tahoma;
+  margin-left: 110px;
+}
+.portalnavbar a {
+  text-decoration: none;
+  color: white;
+}
+.clearfix:after {
+  visibility: hidden;
+  display: block;
+  font-size: 0;
+  clear: both;
+  content: ""
+  height: 0;
+}
+.hireemployee {
+  float: left;
+  background-color: #56a748;
+  color: white;
+  font-family: Tahoma;
+  width: 40%;
+  height: 200px;
+  font-size: 80%;
+  padding: 30px;
+  margin-left: 10%;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+   box-sizing: border-box;
+}
+.sendbutton {
+  background-color: #56a748;
+  color: white;
+  margin-left: 190px;
+  font-family: Tahoma;
+  padding: 10px;
+  border-color: whie
+}
+.joblooking {
+  float: left;
+  background-color: #313131;
+  color: white;
+  font-family: Tahoma;
+  width: 40%;
+  height: 200px;
+  font-size: 80%;
+  padding: 30px;
+  margin-right: 10%;
+  -moz-box-sizing: border-box;
```

Comment:
because you repeat this in multiple classes, you could probably make these another class on its own!